triton-proxy/internal/server/triton.go

```diff
@@ -18,7 +18,10 @@ func buildEnsembleGenerateRequest(req *v1.CreateChatCompletionRequest) *ensemble
 	// this to a template.
 	input := "<|begin_of_text|>"
 	for _, m := range req.Messages {
-		input += fmt.Sprintf("<|start_header_id|> %s <|end_header_id|>\n %s '\n<|eot_id|>\n", m.Role, m.Content)
+		for _, c := range m.Content {
+			// TODO(kenji): Support other types.
+			input += fmt.Sprintf("<|start_header_id|> %s <|end_header_id|>\n %s '\n<|eot_id|>\n", m.Role, c.Text)
```

Comment:
This has not been changed in this PR, but is the `'` before `eot_id` a template syntax?

Reply:
nice catch! this looks like a bug. let me create a separate PR to fix.